.env.sample

```
@@ -4,15 +4,12 @@ POSTGRES_PASSWORD=
 REDIS_PORT=
 
 KAFKA_BOOTSTRAP_SERVERS=
-KAFKA_PORT=
 KAFKA_UI_PORT=
-ZOOKEEPER_PORT=
 
+SPRING_PROFILES_ACTIVE=
 
-SPRING_PROFILES_ACTIVE=
\ No newline at end of file
+OPENAI_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)